lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -237,6 +239,10 @@ def instructor_dashboard_2(request, course_id):  # lint-amnesty, pylint: disable
 
     certificate_invalidations = CertificateInvalidation.get_certificate_invalidations(course_key)
 
+    # x block asides enabled, included rapid response tab
+    if XBlockAsidesConfig.current().enabled:
```

Comment:
@pdpinch We've been adding the `Rapid Responses` tab in the instructor dashboard based on a condition if `xBlockAside is added/enabled`. Should I keep it like that or should I do it like [something similar](https://github.com/mitodl/edx-platform/pull/274/commits/66c92ec6487b6148a3a2e510f41e187b91abb5d2#diff-0d1ee66fb84bee6c3517980a7cefd74a050ccb60da9c0c3b662ab0a2d80d17c0R151-R152) as we did for canvas integration e.g. `Show the canvas tab if the plugin is not installed - (We removed a feature flag there)`?

Reply:
Yes, I think that's a better pattern and it would be good to be consistent.

Reply:
Thanks. Updated my message above, I meant `Show the canvas tab if the plugin is installed`. But you already got it right I believe.